prbt_support/launch/robotmodel.rviz

```diff
@@ -39,32 +39,16 @@ Visualization Manager:
         Expand Link Details: false
         Expand Tree: false
         Link Tree Style: Links in Alphabetic Order
-        prbt_flange:
-          Alpha: 1
-          Show Axes: false
-          Show Trail: false
-          Value: true
-        prbt_gripper_finger_left_link:
-          Alpha: 1
-          Show Axes: false
-          Show Trail: false
-          Value: true
-        prbt_gripper_finger_left_link_bracket:
+        prbt_base:
           Alpha: 1
           Show Axes: false
           Show Trail: false
-          Value: true
-        prbt_gripper_finger_right_link:
```

Comment:
Why do we drop them? Because they are now part of a different package?

Reply:
exactly. Trying to launch `test_urdf.launch` from kinetic-devel failed.